packages/eas-cli/src/graphql/queries/EnvironmentVariablesQuery.ts

```diff
@@ -0,0 +1,130 @@
+import { print } from 'graphql';
+import gql from 'graphql-tag';
+
+import { ExpoGraphqlClient } from '../../commandUtils/context/contextUtils/createGraphqlClient';
+import { withErrorHandlingAsync } from '../client';
+import {
+  EnvironmentVariableEnvironment,
+  EnvironmentVariableFragment,
+  EnvironmentVariablesByAppIdQuery,
+} from '../generated';
+import { EnvironmentVariableFragmentNode } from '../types/EnvironmentVariable';
+
+export const EnvironmentVariablesQuery = {
+  async byAppIdWithSensitiveAsync(
+    graphqlClient: ExpoGraphqlClient,
+    {
+      appId,
+      environment,
+      filterNames,
+    }: {
+      appId: string;
+      environment: EnvironmentVariableEnvironment;
+      filterNames?: string[];
+    }
+  ): Promise<{
+    appVariables: EnvironmentVariableFragment[];
+  }> {
```

Comment:
It's not used anywhere, is it? If it's not and we are dropping sudo requirement, we can delete this I guess and just use standard env vars query?

Reply:
ditto